tests/test_embedding.py

```diff
@@ -1,24 +1,69 @@
 from conftest import get_metrics
-from pyriemann.embedding import Embedding
+import numpy as np
+from pyriemann.embedding import (SpectralEmbedding,
+                                 LocallyLinearEmbedding,
+                                 barycenter_weights,
+                                 riemann_lle)
 import pytest
 
 
```

Comment:
Hmja, I wrote up something pretty dumb which would at least indicate if the embeddings didn't work at all:
embed separated blobs in one dimension and do a nearest centroid classification where training and test data are the same.
But it could be that for some random samples this classification fails and then there's an error in the tests that's not really an error. This should be mitigated by choosing a very high class separation but one can neverr be sure.